schedule/yast/yast_hostname/yast_hostname+dhcp_hostname_aarch64.yaml

```diff
@@ -0,0 +1,23 @@
+---
+name: yast_hostname+dhcp_hostname
+description: >
+  Test suite uses default option to set dhcp (using ifgf=*=dhcp), hostname.
+  Test validates if installation can successfully start in case of usage of
+  these parameters.
+vars:
+  EXIT_AFTER_START_INSTALL: '1'
+  EXPECTED_INSTALL_HOSTNAME: dhcphostname
+  EXTRABOOTPARAMS: ifcfg=*=dhcp
+  NICTYPE_USER_OPTIONS: hostname=dhcphostname
+  YUI_REST_API: 1
+schedule:
+  local_user: []
+  booting: []
+  installation_settings: []
+  installation: []
+  installation_logs: []
+  confirm_reboot: []
+  grub: []
+  first_login: []
+  system_validation:
+    - installation/hostname_inst
```

Comment:
this is not really a system validation, this is for "running system validation", but also another reason is that we will create a feature to clear those array forward, so instead of this if you could please append this test module in the overwrite `clock_and_timezone` would be better to refactor.

Reply:
I have updated ' clock_and_timezone:
    - installation/clock_and_timezone/accept_timezone_configuration
    - installation/hostname_inst'